### PR TITLE
Update threat-analytics.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details about other tabs missing from Threat analytics dashboard

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7407

File: microsoft-365/security/defender-endpoint/threat-analytics.md [hunk 1]
```diff
@@ -63,59 +63,155 @@ Select a threat from the dashboard to view the report for that threat.
 
 ![Image of a threat analytics dashboard.](images/ta_dashboard.png)
 
+## View the threat analytics dashboard
+
+The threat analytics dashboard ([security.microsoft.com/threatanalytics3](https://security.microsoft.com/threatanalytics3)) highlights the reports that are most relevant to your organization. It summarizes the threats in the following sections:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/threat-analytics.md]:
```suggestion
The [threat analytics dashboard](https://security.microsoft.com/threatanalytics3) highlights the reports that are most relevant to your organization. It summarizes the threats in the following sections:
```

File: microsoft-365/security/defender-endpoint/threat-analytics.md [hunk 2]
```diff
@@ -63,59 +63,155 @@ Select a threat from the dashboard to view the report for that threat.
...
+
+- **Latest threats**—lists the most recently published or updated threat reports, along with the number of active and resolved alerts.
+- **High-impact threats**—lists the threats that have the highest impact to your organization. This section lists threats with the highest number of active and resolved alerts first.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/threat-analytics.md]:
```suggestion
- **High-impact threats**—lists the threats that have the highest impact on your organization. This section lists threats with the highest number of active and resolved alerts first.
```

File: microsoft-365/security/defender-endpoint/threat-analytics.md [hunk 3]
```diff
@@ -63,59 +63,155 @@ Select a threat from the dashboard to view the report for that threat.
...
+- **Threat summary**—provides the overall impact of all tracked threats by showing the number of threats with active and resolved alerts.
+
+Select a threat from the dashboard to view the report for that threat.
+
+![Screenshot of threat analytics dashboard.](../../media/threat-analytics/ta_dashboard_mtp.png)
+
+_Threat analytics dashboard. You can also click the Search icon to key in a keyword related to the threat analytics report that you'd like to read._
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/threat-analytics.md]:
```suggestion
_Threat analytics dashboard. You can also select the **Search** icon to key in a keyword related to the threat analytics report that you would like to read._ 
```

File: microsoft-365/security/defender-endpoint/threat-analytics.md [hunk 4]
```diff
@@ -63,59 +63,155 @@ Select a threat from the dashboard to view the report for that threat.
...
-#### Assess the impact to your organization
+#### Assess impact on your organization
 
 Each report includes charts designed to provide information about the organizational impact of a threat:
 
-- **Devices with alerts**: Shows the current number of distinct devices that have been impacted by the threat. A device is categorized as **Active** if there is at least one alert associated with that threat and **Resolved** if *all* alerts associated with the threat on the device have been resolved.
-- **Devices with alerts over time**: Shows the number of distinct devices with **Active** and **Resolved** alerts over time. The number of resolved alerts indicates how quickly your organization responds to alerts associated with a threat. Ideally, the chart should be showing alerts resolved within a few days.
+- **Related incidents**—provides an overview of the impact of the tracked threat to your organization with the following data:
+  - Number of active alerts and the number of active incidents they are associated with
+  - Severity of active incidents
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/threat-analytics.md:105]:
```suggestion
  - Number of active alerts and the number of active incidents they are associated with.
  - Severity of active incidents.
```

File: microsoft-365/security/defender-endpoint/threat-analytics.md [hunk 5]
```diff
@@ -63,59 +63,155 @@ Select a threat from the dashboard to view the report for that threat.
...
+
 ### Mitigations: Review list of mitigations and the status of your devices
 
 In the **Mitigations** section, review the list of specific actionable recommendations that can help you increase your organizational resilience against the threat. The list of tracked mitigations includes:
 
-- **Security updates**: Deployment of security updates or patches for vulnerabilities
-- **Microsoft Defender Antivirus settings**
-  - Security intelligence version
-  - Cloud-delivered protection
+- **Security updates**—deployment of supported software security updates for vulnerabilities found on onboarded devices
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/threat-analytics.md]:
```suggestion
- **Security updates**—deployment of supported software security updates for vulnerabilities found on onboarded devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@DulceMontemayor Please merge this PR, Thanks

**Comment by DulceMontemayor** [comment]
Hi @yogkumgit! I've to check with the PMs. As far as i know, the TA view between Microsoft Defender for Endpoint (MDE) and Microsoft 365 Defender (M365) is different. Hence the difference even in the mitigation tabs that a customer looking at the MDE portal will see compared to what a customer looking at the M365D portal will see

**Comment by yogkumgit** [comment]
@msbemba Please reply/action

**Comment by DulceMontemayor** [comment]
Hi @yogkumgit! Good day! I've just received feedback from my SME (he got sick, hence the delayed reply) and he confirmed that the Microsoft Defender for Endpoint (MDE) is different from the Microsoft 365 Defender (M365) experience. Hence, the tabs that they will see there for the capabilities are also different. MDE is device centric. M365 D converges not just device, but also email detection and protection capabilities.

That said, we should keep the current documentation as it is since it is accurate. We should _not_ merge your recommended updates through because it is incorrect. 

I hope that helps. Thanks!

**Pull request closed without merging**